starter-repos/app-template/config/base/redis/redis_manifest.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   name: redis-master
-  namespace: default
```

Comment:
@Liujingfang1 we believe that namespace shouldn't be specified in the redis configs. Please confirm if this make sense.

Reply:
Correct. Let me fix it.

Reply:
SG. I'll merge this PR given that the dev environment takes the namespace specified in `kubens`